.adr/001-dashboard-v1-architecture.md

```diff
@@ -0,0 +1,88 @@
+# Dashboard V1 architecture
+
+* Status: accepted
+* Deciders: @hanbyul-here @michaelgsuttles @danielfdsilva @nerik 
+* Date: 2023/01/30
+
+Technical Story: https://github.com/NASA-IMPACT/veda-architecture/issues/162
+
+## Context and Problem Statement
+
+The implementation of the [dashboard V1](https://www.figma.com/file/SA0u0sVdw5vM7g9ulV5Xgw/Wireframes?node-id=2455%3A32113&t=pG3PMb33Q1a1NmhV-0) requires some changes in the overall architecture.
+
+The data catalog (and comprising search) and the relationships between content types (for example: "discoveries that use this dataset") are the main reasons why an architecture overhaul may be needed.
+
+The current architecture relies on a series of static files and all the relationships and data structures are created during build time via a series of plugins. There are limitations to the available "queries" because such queries need to have been precomputed - meaning that a discrete list of options must be considered beforehand.
+
+In the case of a search functionality it is not possible to know all the options the user will search. To solve this we'd need to have all possible searchable content in memory, which means the user would have to download a lot of content just to enable search.
+
+## Decision Drivers
+
+- Effort and time to implement
+- Support of needed functionalities
+
+## Considered Options
+
+- [1] Maintain current architecture
+- [2] CMS
+- [3] Hybrid
+- [4] Veda Datastore
+
```

Comment:
I forgot to bring this up last time, but 'Replicability of the instance' (Somebody being able to fork veda-config repo and replace the contents with the same dashboard functionality) should be mentioned somewhere. Users will be able to use the search functionality without extra set up with option [1], but not with [2] or [3]

Reply:
Good call! Will add it